model/image.go

```diff
@@ -294,6 +306,15 @@ func (img *Image) Resample(targetBitsPerComponent int64) {
 	img.BitsPerComponent = int64(targetBitsPerComponent)
 }
 
+// ToJBIG2Image converts current image to the core.JBIG2Image image.
+func (img *Image) ToJBIG2Image() (*core.JBIG2Image, error) {
```

Comment:
Maybe we should name this just `ToJBIG2`?

Reply:
This name was based on the `ToGoImage` and I think that `ToJBIG2` may be ambigous. It might mean that the result would be jbig2 encoded image or just jbig2 encoded byte stream. 
I would keep that naming convention, what do you think?

Reply:
Sure, I have no problem with that. I think the situation was a bit trickier in the case of `ToGoImage`, as the struct is already called `Image` so you cannot have a method called `ToImage`, as it's not really suggestive. On the other hand, naming it just `ToGo`, leaving out `Image` does not look very good either. So it was a bit of a compromise there.

In the current situation I was thinking that `ToJBIG2` is good enough, given that the base struct is `Image`, you would expect the method to return another image. Also, the return value is always visible anyway in the method prototype. And it leaves room for another method like `ToJBIG2Stream`. 

However, there's nothing wrong with `ToJBIG2Image`. I was just asking for a second opinion on the matter.